pkg/hubble/testutils/fake.go

```diff
@@ -326,6 +326,18 @@ var NoopEndpointGetter = FakeEndpointGetter{
 	},
 }
 
+type FakeLinkGetter struct{}
+
+func (e *FakeLinkGetter) Name(ifindex uint32) string {
+	return "lo"
+}
+
+func (e *FakeLinkGetter) GetIfNameCached(ifindex int) (string, bool) {
+	return e.Name(uint32(ifindex)), false
```

Comment:
nit: Since this always returns `lo`

```suggestion
	return e.Name(uint32(ifindex)), true
```